node/core/approval-voting/src/lib.rs

```diff
@@ -774,6 +793,162 @@ where
 	Ok(())
 }
 
+#[derive(Eq, PartialEq)]
+enum RecoveryState {
+	Pending,
+	Complete,
+}
+
+impl RecoveryState {
+	fn peek(&self) -> bool {
```

Comment:
I recommend renaming this to `is_complete`

Reply:
Removed recovery state entirely now that it is no longer needed.